### DISCUSSION TITLE
same codebase able connect more than one workspace?

### DISCUSSION THREAD

**Discussion by metroruby** [nrwl/nx#23510] [Questions]
Hi,
I have codebase that connect workspace of NX Cloud A already, also I want to use this codebase(fork repo) to connect with another one workspace of NX Cloud B

**Comment by jaysoo** [comment]
There isn't a way to connect the same token to multiple workspace. You will need to remove `nxCloudAccessToken` in your fork's `nx.json` and run `nx connect` again.
